Packages/com.unity.inputsystem/Documentation~/Devices.md

```diff
@@ -8,7 +8,9 @@
     * [Device creation](#device-creation)
     * [Device removal](#device-removal)
     * [Device resets](#device-resets)
+    * [Device syncs](#device-syncs)
     * [Device enabling and disabling](#device-enabling-and-disabling)
+    * [Backgrouund and focus change behavior](#background-and-focus-change-behavior)
```

Comment:
Typo - please change Backgrouund to Background

Reply:
Fixed